models/delete.py

```diff
@@ -2,11 +2,12 @@
 
 from models.ingest import EncoderEnum
 from models.vector_database import VectorDatabase
+from models.file import File
 
 
 class RequestPayload(BaseModel):
     index_name: str
-    file_url: str
+    files: list[File]
```

Comment:
Why do then need to pass `type` for deleting files? If we only need the URL you should create a new Model for that.